build/toolchain/linux/BUILD.gn

```diff
@@ -89,39 +149,90 @@ gcc_toolchain("clang_x64") {
   is_clang = true
 }
 
-gcc_toolchain("clang_arm64") {
-  if (use_clang_type_profiler) {
-    prefix = rebase_path("//third_party/llvm-allocated-type/Linux_x64/bin",
-                         root_build_dir)
-  } else {
-    prefix = rebase_path("//buildtools/linux-x64/clang/bin", root_build_dir)
+gcc_toolchain("x64") {
+  prefix = ""
+  cc = "${compiler_prefix}${prefix}gcc"
+  cxx = "${compiler_prefix}${prefix}g++"
+
+  readelf = "${prefix}readelf"
+  nm = "${prefix}nm"
+  ar = "${prefix}ar"
+  ld = cxx
+  strip = "${prefix}strip"
+
+  toolchain_cpu = "x64"
+  toolchain_os = "linux"
+  is_clang = false
+}
+
+gcc_toolchain("riscv32") {
```

Comment:
Are these primarily for parity with Dart? Similar question to above re: testing since we don't currently build for these today in Flutter.

/cc @zanderso

Reply:
Yes, Flutter's gcc builds are non-function before and after this CL.